CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 v1.9.2 [unreleased]
--	[#21609](https://github.com/influxdata/influxdb/pull/21609): fix: avoid rewriting fields.idx unnecessarily
+-	[#21634](https://github.com/influxdata/influxdb/pull/21634): fix: group by returns multiple results per group in some circumstances
 
-v1.9.1 [2021-06-03]
+v1.9.1 [unreleased]
```

Comment:
Was the date-change here intentional? vs. leaving 1.9.1 as-is and adding 1.9.3

Reply:
Yes. We are not building an official release of OSS 1.9 so this date was an error.